modules/core/src/main/java/org/apache/ignite/internal/processors/cache/local/atomic/GridLocalAtomicCache.java

```diff
@@ -858,9 +866,6 @@ private Object updateAllInternal(GridCacheOperation op,
         boolean readThrough,
         boolean keepBinary
     ) throws IgniteCheckedException {
-        if (keyCheck)
```

Comment:
Unsafe change. This method could be invoked form `puAll0()` for example. The same for `GridDhtAtomicCache`.

Reply:
I have checked absolutely all callers of this method, and they all have key check upstream. putAll0 has this check in upstream putAll.

I think we never need to repeat it here. I can add javadoc stating that keys need to already be checked.

Reply:
Okay.